modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -497,9 +503,10 @@ TEST(TestAgeGenderIE, CPUConfig)
     std::tie(age, gender) = cv::gapi::infer<AgeGender>(in);
     cv::GComputation comp(cv::GIn(in), cv::GOut(age, gender));
 
-    auto pp = cv::gapi::ie::Params<AgeGender> {
-        model_path, weights_path, device_id
-    }.cfgOutputLayers({ "age_conv3", "prob" }).pluginConfig({{"ENFORCE_BF16", "NO"}});
+    auto pp = cv::gapi::ie::Params<AgeGender> {model_path, weights_path, device_id
+                                              }.cfgOutputLayers({ "age_conv3", "prob" })
+                                               .pluginConfig({{"CPU_THROUGHPUT_STREAMS",
+                                                               "CPU_THROUGHPUT_NUMA"}});
```

Comment:
How it's work ? You pass map with {'key': 'key'}, is it ?
I believe it should be `{CPU_THROUGHPUT_STREAMS: <number>`}, correct me if I'm wrong

Reply:
`CPU_THROUGHPUT_NUMA` is one of the default values that could be passed for this `CPU_THROUGHPUT_STREAMS` key

Reply:
Got it!